src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/integration/search/SearchStaffByEmail.kt

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.integration.search
+
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.db.entities.Staff
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.integration.IntegrationTestBase
+
+class SearchStaffByEmail : IntegrationTestBase() {
+
+  @Test
+  fun `must return staff by partial matched email`(): Unit = runBlocking {
+    val first = staffRepository.save(Staff(firstName = "John", lastName = "Smith", jobTitle = "Probation Practitioner", email = "john.smith@staff.com"))
+    val second = staffRepository.save(Staff(firstName = "Joanne", lastName = "Smith", email = "joanne.smith@staff.com"))
+
+    webTestClient.get()
+      .uri("/staff/search?email=sm")
+      .headers(setAuthorisation())
+      .exchange()
+      .expectStatus()
+      .isOk
+      .expectBody()
+      .jsonPath("$.[?(@.email=='${first.email}')].firstName").isEqualTo(first.firstName)
+      .jsonPath("$.[?(@.email=='${first.email}')].lastName").isEqualTo(first.lastName)
+      .jsonPath("$.[?(@.email=='${first.email}')].jobTitle").isEqualTo(first.jobTitle)
+      .jsonPath("$.[?(@.email=='${second.email}')].firstName").isEqualTo(second.firstName)
+      .jsonPath("$.[?(@.email=='${second.email}')].lastName").isEqualTo(second.lastName)
+  }
+
+  @Test
+  fun `must favour username over domain name`(): Unit = runBlocking {
+    for (i in 0..21) {
```

Comment:
Do we need 21 of these? Will 2 not do just for this test?

This might be covered by a "proper" test data set as mentioned in my other comment, but maybe this test is too specific?!